packages/integration-tests/api/api.js

```diff
@@ -394,6 +394,121 @@ async function addProviderApi({ prefix, provider }) {
   });
 }
 
+/**
+ * Fetch a list of providers from the Cumulus API
+ *
+ * @param {Object} params - params
+ * @param {string} params.prefix - the prefix configured for the stack
+ * @returns {Promise<Object>} - the list of providers fetched by the API
+ */
+async function getProviders({ prefix }) {
+  const payload = await callCumulusApi({
+    prefix: prefix,
+    functionName: 'ApiProvidersDefault',
+    payload: {
+      httpMethod: 'GET',
+      resource: '/providers',
+      path: '/providers'
+    }
+  });
+
+  return payload;
+}
+
+/**
+ * Fetch a provider from the Cumulus API
+ *
+ * @param {Object} params - params
+ * @param {string} params.prefix - the prefix configured for the stack
+ * @param {string} params.providerId - the ID of the provider to fetch
+ * @returns {Promise<Object>} - the provider fetched by the API
+ */
+async function getProvider({ prefix, providerId }) {
+  const payload = await callCumulusApi({
+    prefix: prefix,
+    functionName: 'ApiProvidersDefault',
+    payload: {
+      httpMethod: 'GET',
+      resource: '/providers/{providerId}',
+      path: `/providers/${providerId}`,
+      pathParameters: {
+        id: providerId
+      }
+    }
+  });
+
+  return payload;
+}
+
+/**
+ * Fetch a list of collections from the Cumulus API
+ *
+ * @param {Object} params - params
+ * @param {string} params.prefix - the prefix configured for the stack
+ * @returns {Promise<Object>} - the list of collections fetched by the API
+ */
+async function getCollections({ prefix }) {
+  const payload = await callCumulusApi({
```

Comment:
This assignment is not necessary. Could just be return `callCumulusApi({`

Reply:
changed